server/lib/schema/index.ts

```diff
@@ -1159,6 +1180,124 @@ const schema = new GraphQLSchema({
           },
         ),
       },
+      concedeToMalicious: {
+        type: GraphQLBoolean,
+        args: {
+          id: { type: GraphQLString },
+        },
+        resolve: requireUser(
+          "You must be logged in to decline to challenge a workspace",
+          async (obj, { id, input }, context) => {
+            const workspace = await Workspace.findByPk(id);
+            if (workspace === null) {
+              return false;
+            }
+
+            await workspace.update({
+              isAwaitingHonestExpertDecision: false,
+            });
+
+            const parentWorkspace = await Workspace.findByPk(
+              workspace.parentId,
+            );
+
+            if (parentWorkspace === null) {
+              return false;
+            }
+
+            const grandParentWorkspace = await Workspace.findByPk(
+              parentWorkspace.parentId,
+            );
+
+            if (grandParentWorkspace === null) {
+              return false;
+            }
+
+            const judgeBlocks = (await workspace.$get("blocks")) as Block[];
+            const judgeQuestion = judgeBlocks.find(b => b.type === "QUESTION");
+
+            const maliciousBlocks = (await parentWorkspace.$get(
+              "blocks",
+            )) as Block[];
+
+            const maliciousQuestion = maliciousBlocks.find(
+              b => b.type === "QUESTION",
+            );
+
+            // make honest response field equal to malicious answer candidate
+            const honestBlocks = (await grandParentWorkspace.$get(
+              "blocks",
+            )) as Block[];
+
+            const grandParentAnswerDraft = honestBlocks.find(
+              b => b.type === "ANSWER_DRAFT",
+            );
+
+            if (grandParentAnswerDraft && maliciousQuestion && judgeQuestion) {
+              const newAnswerDraftValue = extractMaliciousAnswerValueFromMaliciousQuestionAndJudgeQuestion(
+                maliciousQuestion.value,
+                judgeQuestion.value,
+              );
+
+              await grandParentAnswerDraft.update({
+                value: newAnswerDraftValue,
+              });
+            }
+
+            // mark honest as resolved
+            // and check to see if upstream judge should be marked as stale
+            await grandParentWorkspace.update({ isCurrentlyResolved: true });
+            if (grandParentWorkspace.parentId) {
+              const upstreamJudge = await Workspace.findByPk(
+                grandParentWorkspace.parentId,
+              );
+              if (upstreamJudge === null) {
+                return false;
+              }
+              const children = (await upstreamJudge.$get(
+                "childWorkspaces",
+              )) as Workspace[];
+              let allResolvedOrArchived = true;
+              for (const child of children) {
+                if (!(child.isCurrentlyResolved || child.isArchived)) {
```

Comment:
Would prefer `if (!child.isCurrentlyResolved  || !child.isArchived)` to reduce mental gymnastics (and save 1 character ;) )

Reply:
Sure. If you distribute `!` over disjunction though it turns into conjunction, so will be:

`if (!child.isCurrentlyResolved && !child.isArchived)`

Reply:
Good catch. Noticed this is unchanged thus far, but I marked the PR as LGTM because I figure this is a quick change that we don't need to go back-and-forth on.

Reply:
Never mind about unchanged, it turns out that going through `View changes` shows the outdated version of the code.
![image](https://user-images.githubusercontent.com/1892071/63126970-0ac89e00-bf66-11e9-81e1-282ce6381b73.png)